mapper/amount.go

```diff
@@ -27,3 +28,50 @@ func FeeAmount(value int64) *types.Amount {
 func AvaxAmount(value *big.Int) *types.Amount {
 	return Amount(value, AvaxCurrency)
 }
+
+func Erc721Amount(
+	indexHash common.Hash,
+	contractAddress common.Address,
+	contractSymbol string,
+	contractDecimal uint8,
+	isSender bool) *types.Amount {
+	index := indexHash.Big()
+	metadata := make(map[string]interface{})
+	metadata[TokenTypeMetadata] = "ERC721"
+	metadata[ContractAddressMetadata] = contractAddress.String()
+	metadata[IndexTransferedMetadata] = index.String()
+
+	return &types.Amount{
```

Comment:
IMO this doesn't make sense to have an amount. You should just return metadata.

Reply:
Unused function from before our discussion.  Deleting